galley/pkg/crd/validation/monitoring.go

```diff
@@ -42,6 +42,14 @@ var (
 		Name: "galley_validation_http_error",
 		Help: "Resource validation http serve errors",
 	}, []string{"status"})
+	metricWebhookConfigurationUpdateError = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Can you check with Doug that the metrics are pulled by prmetheus and have some graph ?

Reply:
These metrics show up in Prometheus, but they aren't in any of our dashboards. 

@douglas-reid, do we provide istio-specific metrics in our dashboards?

Reply:
we have istio-specific metrics in our dashboards, yes. The dash for Mixer, for instance, shows some of the self-monitoring stats from Mixer. We could add the following to the Pilot dash or create a new one.